### ISSUE TITLE
Add support for `disabled` property in react-draggable

### ISSUE THREAD

**Issue by p4bloch** [bokuweb/react-rnd#125]
### Feature request

In my use case I would like to have the enhanced component to be able to have all its interaction disabled (dragging, resizing) depending on a boolean flag, like an edit/view mode feature.

I see there is a disabled property in `react-draggable`, but in this library it is not exposed. I did some testing locally and it works as expected if we add the prop, but wanted to raise an issue before making a PR to confirm whether you agree with this.

Basically I propose adding this 👇

```diff
      <Draggable
         {...}
         onDrag={this.onDrag}
         onStop={this.onDragStop}
         axis={this.props.dragAxis}
+        disabled={this.props.disableDragging}
         grid={this.props.dragGrid}
         bounds={this.props.bounds ? this.state.bounds : undefined}
       >
```

What do you think? Thanks! :)

**Comment by bokuweb** [comment]
@p4bloch Thanks for your request!! It's a nice request. I'm gonna append `disableDragging`!!

**Comment by p4bloch** [comment]
@bokuweb care to point me somewhere I can read how to get the local environment working? I tested this by linking to my project but it would be best to do it the way you develop this library (and maybe start a contribs guide doc?). Thanks!

**Comment by bokuweb** [comment]
@p4bloch Thanks! I'm gonna append dev and contribute guide to readme... 🙏  Please run `npm run build:example-watch` and open `docs/index.html` in your browser.If you want to change example, please modify `docs/src/example.js`.

**Comment by p4bloch** [comment]
Here's my proposal: [Diff link](https://github.com/bokuweb/react-rnd/compare/master...p4bloch:125-disable-dragging?expand=1). Waiting on that contrib guide to know how to proceed with the PR (tests, dist included or not, etc). Feel free to comment 😄

**Comment by bokuweb** [comment]
@p4bloch Thanks. It looks good to me. Since I'll merge, please will you make a PR?We will add the contribution guide later.

**Comment by p4bloch** [comment]
Closing as this was implemented in #126. Thanks for being so helpful @bokuweb :)

**Comment by pionl** [comment]
Hi @bokuweb @p4bloch, thank you for the feature 👍  I've got small issue with the cursor. 

When I construct the rnd with disabled dragging, the cursor style is not set in the div (this is OK - cursor: normal), but when I construct the rnd with`disabledDragging` with `false` and change `disabledDragging` to `true` (my component state change), the cursor won't change (stays cursor: move), UI + functions is disabled.

The `innerStyle` is constructed correctly, but it won't update the wrapper style. On hot reload the cursors changes.

Console.log(innerStyle) on state change.
```
Object {width: "auto", height: "auto", cursor: "normal", display: "inline-block", position: "absolute"…}
```

Outputed HTML

```html
<div class="react-draggable react-draggable-dragged" style="touch-action: none; width: auto; height: auto; cursor: move; display: inline-block; position: absolute; transform: translate(318px, 184px);">
   <div style="position: relative; user-select: auto; width: 388px; height: 111px; box-sizing: border-box;">
      the content
   </div>
</div>
</div>
```

**Comment by bokuweb** [comment]
@p4bloch Thanks for your report. I'll fix it.
